cmd/syncProcessor.go

```diff
@@ -95,6 +97,28 @@ type interactiveDeleteProcessor struct {
 
 	// count the deletions that happened
 	incrementDeletionCount func()
+
+	//dryrunMode
+	dryrunMode bool
+}
+
+// struct represents a removal
+type DeleteTransfer struct {
```

Comment:
Why do we need a new struct? Is storedObject too verbose?

Reply:
Yup, same question. Fields are redundant in `DeleteTransfer` struct.

Reply:
@zezha-msft @mohsha-msft  If I were to use storedObject instead of DeleteTransfer struct I would need to capitalize the fields that we want to json.Marshal() and since storedObject is used throughout the code base I would have to refactor by capitalizing all the references. 
Another alternative to making a DeleteTransfer struct is using the copyTransfer struct to marshal.